src/napari_matplotlib/tests/test_slice.py

```diff
@@ -1,10 +1,14 @@
-import numpy as np
+import pytest
 
 from napari_matplotlib import SliceWidget
 
 
-def test_scatter(make_napari_viewer):
-    # Smoke test adding a histogram widget
+@pytest.mark.mpl_image_compare
+def test_slice(make_napari_viewer, brain_data):
+    # Smoke test adding a slice widget
     viewer = make_napari_viewer()
-    viewer.add_image(np.random.random((100, 100, 100)))
-    SliceWidget(viewer)
+    viewer.add_image(
+        brain_data[0], **brain_data[1]
+    )  # (np.random.random((100, 100, 100)))
```

Comment:
```suggestion
    ) 
```
No need to keep old comments around